### PR TITLE
Tune tmt tests regexes to align with QE automation

### PR DESCRIPTION
**Pull request by fernflower**
In order for upstream tests and those launched with QE tooling to produce the comparable set of results it's a wise thing to do.

### CODE REVIEW

**Comment by fernflower** [comment]
Proof it works:

https://github.com/fernflower/leapp/actions/runs/3436562372

**Comment by fernflower** [comment]
Rerunning on https://github.com/fernflower/leapp-repository/pull/8#issuecomment-1310213150:

/rerun-sst - https://github.com/fernflower/leapp-repository/actions/runs/3436810388
/rerun - https://github.com/fernflower/leapp-repository/actions/runs/3436808968

**Comment by fernflower** [comment]
Final one:
/rerun-sst (only morf tests are run) - https://github.com/fernflower/leapp-repository/actions/runs/3438358137
/rerun (all but morf) - https://github.com/fernflower/leapp-repository/actions/runs/3438870673

**Comment by fernflower** [comment]
Filtered out tier2 and tier3 tests. 
Proof of work:
/rerun - https://github.com/fernflower/leapp-repository/actions/runs/3564701127

/reun-sst - https://github.com/fernflower/leapp-repository/actions/runs/3564702351

File: .github/workflows/tmt-tests.yml [hunk 1]
```diff
@@ -49,45 +57,57 @@ jobs:
...
 
   call_workflow_tests_87to91_integration:
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-8to9.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
       variables: "LEAPP_DEVEL_TARGET_PRODUCT_TYPE=beta;RHSM_SKU=RH00069;TARGET_RELEASE=9.1;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms"
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:90]: This target is no longer beta. Other variables are needed in other sections as well. We'll try to remove these in the mid-term (and include them in plans) but for now I think we have to match the relevant `TF` configuration sections here from downstream tests (composes, variables, regex, context distro, etc).

File: .github/workflows/tmt-tests.yml [hunk 2]
```diff
@@ -49,45 +57,57 @@ jobs:
...
+      && contains(fromJson('["OWNER", "MEMBER", "COLLABORATOR"]'), github.event.comment.author_association)
 
   call_workflow_tests_86to90_integration:
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-8to9.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*morf)"
```
[the remark this answers is not in the document]
Comment (fernflower) [reply .github/workflows/tmt-tests.yml:76]: Good catch!

File: .github/workflows/tmt-tests.yml (hunk 1 above)
Comment (fernflower) [reply .github/workflows/tmt-tests.yml:90]: I will remove beta here, thanks. But what else did I miss? I went through TF/conf files and can't say that I see any discrepancies there.

Comment (mm0ran) [reply .github/workflows/tmt-tests.yml]: On a quick check I see differences in `RHSM_REPOS` which are now EUS (see 86* confs in `TF/conf`).

**Comment by fernflower** [comment]
After mmoran's review, proof of work:

/rerun - https://github.com/fernflower/leapp-repository/actions/runs/3574176230
/rerun-sst - https://github.com/fernflower/leapp-repository/actions/runs/3574177130

File: .github/workflows/tmt-tests.yml [hunk 3]
```diff
@@ -49,55 +59,73 @@ jobs:
...
+  call_workflow_tests_87to90_integration:
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-8to9.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
-      variables: "LEAPP_DEVEL_TARGET_PRODUCT_TYPE=beta;RHSM_SKU=RH00069;TARGET_RELEASE=9.1;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
```
Comment (pirat89) [inline .github/workflows/tmt-tests.yml:93]: @fernflower @mm0ran the same here. eus repos does not exist for odd releases (8.7, ..) and the last release (7.9, 8.10, ...)

**Comment by fernflower** [comment]
Hopefully last /rerun -https://github.com/fernflower/leapp-repository/actions/runs/3704655277

File: .github/workflows/tmt-tests.yml [hunk 4]
```diff
@@ -49,55 +59,73 @@ jobs:
...
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(.*e2e)"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*7to8)(.*e2e)"
       compose: "RHEL-8.6-rhui"
       environment_settings: '{"provisioning": {"post_install_script": "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys"}}'
       pull_request_status_name: "8to9-aws-e2e"
-      variables: "RHUI=aws"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms;RHUI=aws'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:127]:
```suggestion
      variables: 'SOURCE_RELEASE=8.6;TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms;RHUI=aws'
```

File: .github/workflows/tmt-tests.yml [hunk 5]
```diff
@@ -49,55 +59,73 @@ jobs:
...
-  call_workflow_tests_8to9_sst:
+  call_workflow_tests_86to90_sst:
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-8to9.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(.*morf)"
+      tmt_plan_regex: "^(/plans/morf)(?!.*sap)"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:109]:
```suggestion
      variables: 'SOURCE_RELEASE=8.6;TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
```

File: .github/workflows/tmt-tests.yml [hunk 6]
```diff
@@ -49,55 +59,73 @@ jobs:
...
+  call_workflow_tests_87to90_integration:
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-8to9.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
-      variables: "LEAPP_DEVEL_TARGET_PRODUCT_TYPE=beta;RHSM_SKU=RH00069;TARGET_RELEASE=9.1;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:93]:
```suggestion
      variables: 'SOURCE_RELEASE=8.7;TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-rpms,rhel-8-for-x86_64-baseos-rpms'
```

File: .github/workflows/tmt-tests.yml [hunk 7]
```diff
@@ -49,55 +59,73 @@ jobs:
...
 
   call_workflow_tests_86to90_integration:
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-8to9.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*7to8)(?!.*morf)"
+      variables: 'TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:79]:
```suggestion
      variables: 'SOURCE_RELEASE=8.6;TARGET_RELEASE=9.0;TARGET_KERNEL=el9;RHSM_REPOS=rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms'
```

File: .github/workflows/tmt-tests.yml [hunk 8]
```diff
@@ -49,55 +59,73 @@ jobs:
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*8to9)(.*e2e)"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*8to9)(.*e2e)"
       compose: "RHEL-7.9-rhui"
       environment_settings: '{"provisioning": {"post_install_script": "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; echo 42; yum-config-manager --enable rhel-7-server-rhui-optional-rpms"}}'
       pull_request_status_name: "7to8-aws-e2e"
       variables: "RHUI=aws"
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:66]:
```suggestion
      variables: "SOURCE_RELEASE=7.9;TARGET_RELEASE=8.6;RHUI=aws"
```

File: .github/workflows/tmt-tests.yml [hunk 9]
```diff
@@ -10,37 +10,47 @@ jobs:
...
     uses: oamg/leapp/.github/workflows/reuse-tests-7to8.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*8to9)(.*morf)"
-      pull_request_status_name: "7.9to8.4-sst"
+      tmt_plan_regex: "^(/plans/morf)(?!.*sap)"
+      pull_request_status_name: "7.9to8.8-sst"
       update_pull_request_status: 'false'
+      variables: 'TARGET_RELEASE=8.8'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:50]:
```suggestion
      variables: SOURCE_RELEASE=7.9;'TARGET_RELEASE=8.8'
```

File: .github/workflows/tmt-tests.yml [hunk 10]
```diff
@@ -10,37 +10,47 @@ jobs:
...
 
   call_workflow_tests_79to86_integration:
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-7to8.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*morf)"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*morf)"
       variables: 'TARGET_RELEASE=8.6'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:34]:
```suggestion
      variables: 'SOURCE_RELEASE=7.9;TARGET_RELEASE=8.6'
```

File: .github/workflows/tmt-tests.yml [hunk 11]
```diff
@@ -10,37 +10,47 @@ jobs:
...
     needs: call_workflow_copr_build
     uses: oamg/leapp/.github/workflows/reuse-tests-7to8.yml@master
     secrets: inherit
     with:
       copr_artifacts: ${{ needs.call_workflow_copr_build.outputs.artifacts }}
-      tmt_plan_regex: "^(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*morf)"
-      pull_request_status_name: "7.9to8.4"
+      tmt_plan_regex: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*morf)"
+      pull_request_status_name: "7.9to8.8"
+      variables: 'TARGET_RELEASE=8.8'
```
Comment (mm0ran) [inline .github/workflows/tmt-tests.yml:21]:
```suggestion
      variables: 'SOURCE_RELEASE=7.9;TARGET_RELEASE=8.8'
```

**Review by mm0ran — CHANGES_REQUESTED** [review]
Some additional variables from `TF/conf`

**Comment by fernflower** [comment]
@mm0ran Latest commit should add `SOURCE_RELEASE` to all tests scheduled.

Here will be the /rerun results - https://github.com/fernflower/leapp-repository/actions/runs/3712314197

**Review by mm0ran — APPROVED** [review]

**Comment by mm0ran** [comment]
LGTM!

**Pull request merged**